test/fixtures/googleembeds.json

```diff
@@ -6,7 +6,7 @@
       "children": [
         {
           "type": "text",
-          "value": "Hello, Google Docs Embed! "
+          "value": "Hello, Google Docs Embed!"
```

Comment:
Slightly different handling of trailing spaces. shouldn't affect HTML generation.

Reply:
I think it's fine.